packages/shared/src/node-apis/redis.ts

```diff
@@ -120,58 +114,47 @@ const accountFee = z
   })
   .transform(({ account, bps }) => ({ account, commissionBps: bps }));
 
-const vaultDepositSchema = (network: ChainflipNetwork) =>
-  jsonString.pipe(
-    z
-      .object({
-        amount: u128,
-        destination_address: z.union([ethereumAddress, solanaAddress, btcAddress(network)]),
```

Comment:
changed the address to accept any string to simplify the schema. nothing we can do if the ingress-egress-tracker puts an invalid address into redis anyway